theano/misc/tests/test_gnumpy_utils.py

```diff
@@ -44,9 +40,6 @@ def test2(shape=(3, 4, 5)):
     Make sure that the gnumpy conversion is exact from CudaNdarray to
     garray back to CudaNdarray.
     """
-    gpu = theano.sandbox.cuda.basic_ops.gpu_from_host
```

Comment:
The removed 3 lines, which are defining function `ii`, are not used in function `test2`.